examples/05_tool_inference_lock/ex_tool_inference_lock/main.rb

```diff
@@ -129,8 +129,6 @@ def onUserText(text, view)
         end
 
         view.invalidate
-      rescue ArgumentError
-        Ui.messagebox('Invalid length')
```

Comment:
What's this change for?

Reply:
Mistake from previous PR. This rescue was supposed to be moved from the method as a whole to the Length parsing specifically, but I forgot to remove this code.

This should probably not be squashed with the rest as it is an unrelated change.